app/api/dao/user.py

```diff
@@ -142,20 +143,33 @@ def get_user_by_username(username):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id, search_query="", is_verified=None):
+    def list_users(user_id, search_query='', is_verified=None, name_asc='', name_desc=''):
```

Comment:
Can we not have one variable that checks for this? sort_order = 1 for ascending, -1 for descending, else no explicit sorting?

Reply:
Yes, we can use 1 variable for the sorting, but I follow the name parameter as describe in the issue like name_asc for sorting for ascending according to the name and vice-versa.
I think it will be helpful if we want to sort via the creation date. In that case, another parameter can be added like creation_asc / creation_desc.
Please, let me know what will be the good option @isabelcosta @ramitsawhney27 Do I need to need to change it or go with name_asc / name_desc?

Reply:
Add another parameter that takes the field to sort on then, rather than hardcoding the name of the field in the API parameter

Reply:
Ok, I change it